src/encoded/static/components/auditmatrixv1.js

```diff
@@ -0,0 +1,740 @@
+import React from 'react';
```

Comment:
This the one we are going to delete (next sprint) right?

Reply:
Deleted next sprint, yes.